atd-vze/src/views/Crashes/PeopleDetailsCard.js

```diff
@@ -16,19 +16,10 @@ import { primaryPersonDataMap, personDataMap } from "./peopleDataMap";
 import { GET_PERSON_LOOKUPS } from "../../queries/lookups";
 import { UPDATE_PERSON } from "../../queries/people";
 
-const PeopleDetailsCard = ({
-  data,
-  isExpanded,
-  toggleAccordion,
-  refetch,
-  error,
-  loading,
-  ...props
-}) => {
-  const { data: lookupSelectOptions } = useQuery(GET_PERSON_LOOKUPS);
+const PeopleDetailsCard = ({ data, refetch, ...props }) => {
+  const [isOpen, setIsOpen] = useState(true);
 
-  if (loading) return "Loading...";
-  if (error) return `Error! ${error.message}`;
+  const { data: lookupSelectOptions } = useQuery(GET_PERSON_LOOKUPS);
```

Comment:
i think there is a hidden race condition here where if you tried to edit a person or unit record before the lookups query resolves, you would hit an error.  i'm not sure if we want to block rendering of the related records table until we have the lookup values, or handle this further down the tree where they are used by the related records table component 🤷 

either way, i think it would be good to add some kind of handler to prevent this edge case where the lookups query fails or is low. i guess you could also move the lookups queries into the single mega query we use for the crash data 😈

Reply:
i think it makes sense to handle it by blocking rendering of related records until the query is finished?